commands.js

```diff
@@ -286,6 +286,48 @@ var commands = exports.commands = {
 	},
 	makechatroomhelp: ["/makechatroom [roomname] - Creates a new room named [roomname]. Requires: ~"],
 
+	personalroom: 'makegroupchat',
+	makegroupchat: function (target, room, user, connection, cmd) {
+		// First and foremost, check Resource Monitor's rate limit
+		if (ResourceMonitor.countGroupChat(connection.ip)) {
+			connection.popup("Due to high load, you are limited to creating 4 group chats every hour.");
+			return;
+		}
+
+		var targets = target.split(',');
+		// Check if the custom part of the room title is empty. It needs a title.
+		var roomName = targets[0] || Math.floor(Math.random() * 100000000);
+		// Chosen name have the same limitations as regular rooms.
+		if (roomName.includes(',') || roomName.includes('|') || roomName.includes('[') || roomName.includes('-')) {
```

Comment:
This crashes if `roomName` is an integer (i.e. if the command is called without any arguments)